modules/registry-common-terms.adoc

```diff
@@ -9,10 +9,15 @@
 This glossary defines the common terms that are used in the registry content.
 
 container::
-Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers in data center, a public or private cloud, or your local host.
+Lightweight and executable images that consist of software and all its dependencies. Because containers virtualize the operating system, you can run containers in a data center, a public or private cloud, or your local host.
 
 Image Registry Operator::
+ifdef::openshift-rosa-hcp[]
```

Comment:
This line and line 18 are in the wrong place. I think this section needs to be like so:
```suggestion
ifdef::openshift-rosa, openshift-rosa-hcp[]
Image Registry Operator::
ifdef::openshift-rosa-hcp[]
The Image Registry Operator runs in the `CONTROL_PLANE_NAMESPACE` of the management cluster, and manages the registry instance in openshift-image-registry of the cluster.
endif::openshift-rosa-hcp[]
ifdef::openshift-rosa[]
The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location.
endif::openshift-rosa[]
endif::openshift-rosa,openshift-rosa-hcp[]
```

Reply:
got it thanks